apps.awesim.org/apps/myjobs/templates/Basic_Schrodinger_Serial_Job_Owens/pbs_script.sh

```diff
@@ -0,0 +1,33 @@
+#PBS -N ondemand/sys/myjobs/basic_schrodinger_serial
+#PBS -j oe
+#PBS -l walltime=0:10:00
+#PBS -l nodes=1:ppn=28
+#PBS -l software=glide+1
+#PBS -S /bin/sh
+qstat -f $PBS_JOBID
+export
+
+#   A Basic Schrodinger Serial Job for the OSC Owens Cluster
+#   https://www.osc.edu/resources/available_software/software_list/schrodinger
+
+#
+# The following lines set up the Schrodinger environment
+#
+module load schrodinger
+#
+# Move to the directory where the job was submitted
+#
+cd $PBS_O_WORKDIR
+pbsdcp -p /users/appl/srb/workshops/compchem/schrodinger/receptor_glide.in $TMPDIR
+cd $TMPDIR
+host=`cat $PBS_NODEFILE|head -1`
+nproc=`cat $PBS_NODEFILE|wc -l`
+#
+# Run schrodinger
+#
+glide -WAIT -HOST ${host}:${nproc} receptor_glide.in
+ls -l
+#
+# Now, copy data (or move) back once the simulation has completed
+#
+pbsdcp -p '*' $PBS_O_WORKDIR
```

Comment:
I had not heard of `pbsdcp` until I saw this command. Given that our audience for these templates are new HPC users it might be worth throwing a link to the online manpage: https://www.osc.edu/~troy/pbstools/man/pbsdcp.

Reply:
```
#
# Move to the directory where the job was submitted
# More information about pbsdcp: https://www.osc.edu/~troy/pbstools/man/pbsdcp
#
``` 
is added to line 16 before usign `pbsdcp`